src/Bot.js

```diff
@@ -14,18 +17,28 @@ class Bot {
             this.channels.get(channel).updated = date;
         }
     }
+
     /**
      * Configure a new channel
      * @param  {String}   channel  Channel ID
      * @param  {Number}   delay    Time between bumps in seconds
      * @param  {Function} callback Callback to execute upon revival
+     * @param  {Array}    tags     Tags to considered relevant
      * @throws When refused
      */
-    configure(channel, delay, callback) {
+    configure(channel, delay, callback, tags = []) {
         if(delay < 10){
             throw `That's too often! Choose a time over 10 seconds.`;
         }
-        this.channels.set(channel, {callback, delay, updated: new Date()});
+        this.channels.set(channel, {callback, delay, updated: new Date(), tags: tags});
```

Comment:
This can just be `tags` instead of `tags: tags`

Reply:
Oh nice